Alignment/APEEstimation/python/MisalignmentScenarios_cff.py

```diff
@@ -1238,3 +1238,30 @@
         ),
     ),
 )
+
+MisalignmentScenario_PhaseI_PseudoAsymptotic = cms.PSet(
```

Comment:
Sure, can be done!
I think also the configuration file should be moved away from APEEstimation to TrackerAlignment.
Will do that to clean things up a bit.